README.md

```diff
@@ -31,6 +31,7 @@ These guidelines are designed exclusively to help the Working Group achieve its
 The Security Working Group has a few communication channels:
 * ROS Discourse using the [wg-security tag](https://discourse.ros.org/tags/wg-security)
 * Chat using a [Matrix room](https://matrix.to/#/!LcRLnAIRWjSCfZmMeD:matrix.org?via=matrix.org)
+* Join the [ROS Security Working Group](https://groups.google.com/forum/#!forum/ros-security) on google groups, which is used as a point of contact to reach the WG (e.g. when we get pinged on GitHub).
```

Comment:
```suggestion
* Join the [ROS Security Working Group](https://groups.google.com/forum/#!forum/ros-security) on google groups, which is used as a point of contact to reach the WG (e.g. when we get mentioned on GitHub using @ros-security-wg).
```

Pushing the previous suggestion a little further: why not include the way to tag us on Github as well?
Not sure if it deserves its own bullet so I left it as part of this one

Reply:
Not sure why I didn't think of that :facepalm: . Thanks @mikaelarguedas!